src/server.py

```diff
@@ -373,17 +385,50 @@ def play_sound(self, sound_event_id, connection, sender, path, iface,
                 self._DBUS_UNKNOWN_SOUND_EVENT_ID,
                 "sound event with id %s does not exist" % sound_event_id)
             return
-        self._cancel_countdown()
-        self.hold()
 
-        uuid_ = str(uuid.uuid4())
-        metadata = self.metadata[sound_event_id]
-        self.players[uuid_] = HackSoundPlayer(uuid_, metadata, sender, options)
-        self.players[uuid_].connect("eos", self.__player_eos_cb, uuid_)
-        self.players[uuid_].connect("error", self.__player_error_cb, uuid_,
-                                    connection, path, iface)
-        self.players[uuid_].play()
-        invocation.return_value(GLib.Variant('(s)', (uuid_, )))
+        # Insert players organized by sound event id.
+        if sound_event_id not in self._uuid_by_event_id:
+            self._uuid_by_event_id[sound_event_id] = set()
+
+        overlap_behavior =\
+            self.metadata[sound_event_id].get("overlap-behavior", "overlap")
```

Comment:
Question, do we want this to be overridable by the user and have `metadata` to be the default behaviour?